mne/source_estimate.py

```diff
@@ -3470,7 +3470,7 @@ def _prepare_label_extraction(stc, labels, src, mode, allow_empty, use_sparse):
             # Efficiency shortcut: use linearity early to avoid redundant
             # calculations
             elif mode == "mean":
-                vertidx = sparse.csr_matrix(vertidx.mean(axis=0))
+                vertidx = sparse.csr_array(vertidx.mean(axis=0)[np.newaxis])
```

Comment:
(nitpick) Is `keepdims` an option here (instead of newaxis)? That feels cleaner to me for some reason... But not sure if our min version includes it

Reply:
Indeed my first move was to try `keepdims` but it didn't work for some reason (maybe not on old SciPy or whatever?)